cheatsheets/Password_Storage_Cheat_Sheet.md

```diff
@@ -15,44 +15,33 @@ This Cheat Sheet provides guidance on the various areas that need to be consider
 
-Encryption should only be used in edge cases where it is necessary to be able to obtain the original password. Some examples of where this might be necessary are:
+**Encryption is a two-way function**, meaning that the original plaintext can be retrieved. Encryption is appropriate for storing data such as a user's address since this data is displayed in plaintext on the user's profile. Hashing their address would result in a garbled mess.
 
-- If the application needs to use the password to authenticate against an external legacy system that doesn't support SSO.
-- If it is necessary to retrieve individual characters from the password.
+In the context of password storage, encryption should only be used in edge cases where it is necessary to obtain the original plaintext password. This might be necessary if the application needs to use the password to authenticate with another system that doesn't support a modern way to programmatically grant access, such as OpenID Connect (OIDC). Where possible, an alternative architecture should be used to avoid the need to store passwords in an encrypted form.
 
-The ability to decrypt passwords represents a serious security risk, so it should be fully risk assessed. Where possible, an alternative architecture should be used to avoid the need to store passwords in an encrypted form.
-
-This Cheat Sheet is focused on password hashing - for further guidance on encrypting passwords see the [Cryptographic Storage Cheat Sheet](Cryptographic_Storage_Cheat_Sheet.md).
+For further guidance on encryption see the [Cryptographic Storage Cheat Sheet](Cryptographic_Storage_Cheat_Sheet.md).
 
 ### How Attackers Crack Password Hashes
 
-Although it is not possible to "decrypt" password hashes to obtain the original passwords, in some circumstances it is possible to "crack" the hashes. The basic steps are:
-
-- Select a likely candidate (such as "password").
-- Calculate the hash of the input.
-- Compare it to the target hash.
+Although it is not possible to "decrypt" password hashes to obtain the original passwords, in some circumstances it is possible to "crack" the hashes.
 
-This process is then repeated for a large number of potential candidate passwords until a match is found. There are a large number of different methods that can be used to select candidate passwords, including:
+The basic steps are:
 
-- Brute force (trying every possible candidate).
-- Dictionaries or wordlists of common passwords
-- Lists of passwords obtained from other compromised sites.
-- More sophisticated algorithms such as [Markov chains](https://github.com/magnumripper/JohnTheRipper/blob/bleeding-jumbo/doc/MARKOV) or [PRINCE](https://github.com/hashcat/princeprocessor)
-- Patterns or masks (such as "1 capital letter, 6 lowercase letters, 1 number").
```

Comment:
While these might be accurate, they seemed like unnecessary examples for a developer just trying to implement good pw storage